main.py

```diff
@@ -2,6 +2,56 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # Create
```

Comment:
Remove these comments, there is no need for them. The code is self-explicit enough